torch/lib/c10d/reducer.cpp

```diff
@@ -544,6 +627,14 @@ void Reducer::mark_variable_ready(VariableIndex index) {
   // Something is wrong if all variables contained in this bucket replica have
   // already been marked as ready.
   if (replica.pending == 0) {
+    // For static graph, it should not get here. If it does, mostly
```

Comment:
Wondering Is there a concrete example of how the application can reach this state with static_graph training (graph is not static, but concretely what does the model end up doing?), and what is meant by mostly?